src/modules/currency/currencies.ts

```diff
@@ -1,16 +1,33 @@
-import BigNumber from 'bignumber.js';
-import memoize from 'memoize-one';
-
 export interface CurrencyConfig {
   name: string;
   code: string;
   symbol: string;
   modifyParts?: (parts: Intl.NumberFormatPart[]) => Intl.NumberFormatPart[];
+  options:
+    | Intl.NumberFormatOptions
+    | {
+        default: null | Intl.NumberFormatOptions;
```

Comment:
Is this a valid type definition to express `type = Intl.NumberFormatOptions & { lessThenOne: Intl.NumberFormatOptions }`?

Reply:
Otherwise looks like `default` is always null

Reply:
Thanks, fixed by adding "never" to the first type option